en/config_fw/weight_and_altitude_tuning.md

```diff
@@ -0,0 +1,113 @@
+
+
+### Adjust airspeed limits such as minimum airspeed [FW_AIRSPD_MIN](../advanced_config/parameter_reference.md#FW_AIRSPD_MIN), the stall airspeed [FW_AIRSPD_STALL](../advanced_config/parameter_reference.md#FW_AIRSPD_STALL)  and trim airspeed [FW_AIRSPD_TRIM](../advanced_config/parameter_reference.md#FW_AIRSPD_TRIM) according to the weight ratio.
+
+In steady state flight we can demand that lift should equal weight of teh vehicle:
+
+$$ Lift = mg = {1\over{2}} \rho c_A F V^2 $$
+
+rearranging this equation for airspeed gives
+
+$$ V = \sqrt{{2mg}\over{\rho c_A F}}  $$
+
+From this equation we see that if we assume a constant angle of attack (which we generally desire), the vehicle weight affects airspeed with a square root relation.
+Therefore, we scale the vehicle stall airspeed,  minimum airspeed and trim airspeed as follows
+
+$$ V_{stall} = \hat{V}_{stall} * \sqrt{m_{gross} \over{m_{base}}}  $$
+
+$$ V_{min} = \hat{V}_{min} * \sqrt{m_{gross} \over{m_{base}}}  $$
+
+$$ V_{trim} = \hat{V}_{trim} * \sqrt{m_{gross} \over{m_{base}}}  $$
+
+## Compensating for Air Density
+
+PX4 automatically compensates various parameters such as trim throttle or minimum sink rate for changing air density. However, it's important to understand that when the [basic TECS tuning](../config_fw/position_tuning_guide_fixedwing.md#tecs-tuning-altitude-and-airspeed) has been done at non-standard sea level conditions, some of the parameters needs to be back-calculated to calibrated values. In the following we will list each parameter which is compensated based on air density and how you can calculate the calibrated value.
+
+### Maximum Climb Rate  [FW_T_CLMB_MAX](../advanced_config/parameter_reference.md#FW_T_CLMB_MAX)
+As we have seen previously, the maximum climb rate can be formulated as
+
+$$ \dot{h}_{max} = { V * ( Thrust - Drag ) \over{m*g}}  $$
+
+The air density affects the airspeed, the thrust and the drag and modelling this effects is not straight forward. However, we can refer to literature and experience, which suggest that for a propeller airplane the maximum climb rate reduces approximately linear with the air density. Therefore, we can write the maximum climb rate as
+
+$$ \dot{h}_{max} = \hat{\dot{h}} * {\rho_{sealevel} \over{\rho}} K$$
+
+where $\rho_{sealevel}$ is the air density at sea level in the standard atmosphere and K is a scaling factor which determines the slope of the function. Rather than trying to identify this constants, the usual practice in aviation is to specify a service ceiling altitude at which the vehicle is still able to achieve a minimum specified climb rate. 
+
+In PX4 the service ceiling [FW_S_CEILING](../advanced_config/parameter_reference.md#FW_S_CEILING) specifies the altitude in standard atmospheric conditions at which the vehicle is still able to achieve a maximum climb rate of 0.5 m/s at maximum throttle and weight equal to [WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE).
```

Comment:
`FW_S_CEILING` does not appear to exist. Do you mean `FW_SERVICE_CEIL`?

Reply:
@hamishwillee Yes, I corrected the name